src/components/Greeting.jsx

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from 'react';
+
+const Greeting = () => {
+  const url = 'http://127.0.0.1:3000/greetings';
+  const [greeting, setGreeting] = useState('');
+
+  useEffect(() => {
+    fetch(url)
+    .then((response) => response.json())
+    .then((greeting) => setGreeting(greeting.text));
+  
+  }, []);
+
+  return (
+    <h3>{greeting}</h3>
+  )
+}
+
+export default Greeting;
```

Comment:
- [ ] Good job on getting the response from the server but as per requirement, you are supposed to use redux for this. Kindly create a store, an action and a reducer that will connect to your API endpoint to get the random greeting.